coalib/coala.py

```diff
@@ -42,18 +42,39 @@ def main():
             return mode_json(args)
 
         if args.show_bears:
-            from coalib.settings.ConfigurationGathering import (
-                get_filtered_bears)
-
-            local_bears, global_bears = get_filtered_bears(
-                args.filter_by_language, log_printer)
-
-            show_bears(local_bears,
-                       global_bears,
-                       args.show_description or args.show_details,
-                       args.show_details,
-                       console_printer)
-
+            if args.filter_by:
+                from coalib.parsing.Filter import Filter
+                filter = args.filter_by[0]
+                filter_args = args.filter_by[1:]
```

Comment:
I had thought of this, but the issue comes with arg parser. 

> --filter-by language cpp --filter-by aspect License

In case of multiple filter-by available, it only yields results for the later filter. I'll try to fix that.

> --filter-by language cpp&haskell

Currently `And` is not available as operator, but if space separated list is provided, it gives bears with `or` operator.
So, for the following:

> coala -B --filter-by language c python

It gives all the bears that can be used of either C or python or both.

As far as `and` operator is concerned, fixing of multiple `--filter-by` will help.

I've not thought of aliases, at the moment. But we just need to add an entry in the map for that. That's fairly easy to do.